Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -0,0 +1,202 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _k8s_install_openshift_okd:
+
+*****************************
+Installation on OpenShift OKD
+*****************************
+
+OpenShift Requirements
+======================
+
+1. Choose preferred cloud provider (this guide was tested in AWS, Azure & GCP)
+
+2. Read `OpenShift documentation <https://docs.okd.io/latest/welcome/index.html>`_ to find out about provider-specific prerequisites
+
+3. `Get OpenShift Installer <https://github.com/openshift/okd#getting-started>`_
+
+.. note:: It's highly recommended to read the docs, unless you have installed
+   OpenShift in the past. Here are a few notes that you may find useful.
```

Comment:
nit: I think this can just be text rather than a call-out note, unless we specifically want to call it out because we're worried that users would avoid reading the documentation (?)

Reply:
To be honest, I'm not quite sure of these, the OpenShift docs don't make it clear enough and/or easy to find, and I wasted a few hours due to issues with each of these. At the same time, I don't think it should be our job to cover the issues with OpenShift docs being too lengthy and readers being reluctant like myself. So I couldn't really make a call either way. Also, AWS MFA issue is not cover in the OpenShift docs at all. As to formatting, I don't really know, these are kind of extra info, so "note" formatting seemed appropriate. I guess if we had exclamation mark and lightbulb types of notes, I'd use a lightbulb here.

Reply:
The provider-specific notes look like useful things to document, I agree ideally it's not our job here but if there's a couple of extra lines to save our users from hours of pain then keeping them is very reasonable to me.

I think sphinx does provide us a couple of options here, though with a quick search it seems like only notes and warnings. *shrug* Well, you've thought about it, I don't feel super strongly, happy to leave as-is.